app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/BacksplashImage.scala

```diff
@@ -102,7 +102,7 @@ final case class BacksplashGeotiff(
   def readWithCache(z: Int, x: Int, y: Int, context: TracingContext[IO])(
       implicit @cacheKeyExclude flags: Flags): IO[Option[MultibandTile]] = {
     val readTags = tags.combine(Map("zoom" -> z.toString))
-    context.childSpan("cache.read(z, x, y)", readTags) use { childContext =>
+    context.childSpan("cache.read:z_x_y:", readTags) use { childContext =>
```

Comment:
We will have to be a little careful here when adding traces, Xray is particular about the string format used for operations and drops segments if they don't adhere.

Reply:
Is there a way we can test this? Like:

- set up a project pointing to a local-to-the-tests scene (like how gt-contrib at least [used to](https://github.com/geotrellis/geotrellis-contrib/blob/3.16.1-M3/vlm/src/test/scala/geotrellis/contrib/vlm/MosaicRasterSourceSpec.scala#L32))
- make some tile requests against that project where we know it has data
- confirm that the traces... don't error? what happens if the string is bad?

Reply:
If the string is bad it issues a warning in the xray daemon - so we actually don't see it on our end -- we can see that in our logs now if it's happening.

Since we control `XRayTracingContext`'s `childSpan` method I think I'll add sanitization to the operation name there and that _should_ make this less brittle.